packages/zipkin/src/instrumentation/httpServer.js

```diff
@@ -31,17 +31,19 @@ class HttpServerInstrumentation {
     serviceName = tracer.localEndpoint.serviceName,
     host,
     port = requiredArg('port'),
+    supportsJoin = true,
```

Comment:
minimum request is to have this set on the tracer and have the httpServer inherit it. I know it is only used here, but it will help us from establishing a pattern which is at the wrong abstraction